CHANGELOG.md

```diff
@@ -7,11 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
-## [17.0.4] - 2024-04-09
+## [18.0.0] - 2024-04-17
```

Comment:
Please add a note about verifySession returning 401 if it detects multiple access tokens

Reply:
done